src/inspector/calculatedAttributes/errors.ts

```diff
@@ -7,7 +7,7 @@ export enum ErrorReason {
 export type ReasonedError = {
   name: string;
   message: string;
-  reason: ErrorReason;
+  reason?: ErrorReason;
```

Comment:
There is an argument to be made here that if the enum reason is not defined you could add your own error reason enum value. Not necessarily "None", but something relevant to the thrown error.